### ISSUE TITLE
pose parameters

### ISSUE THREAD

**Issue by BigDataHa** [Fanziapril/mvfnet#2]
Thank you for your job.
I found that there are seven pose parameters. What do they mean respectively?

**Comment by BigDataHa** [comment]
i thought it scale, pitch, yaw, roll, tx, ty, tz, but i get the wrong result when I render the model to image

**Comment by ghost** [comment]
what result are you getting, I get a shape.ply in the result folder no texture though

**Comment by BigDataHa** [comment]
I just added pose parameters to the neutral model， But I found the pose parameters is strange

**Comment by Fanziapril** [comment]
Hi, the pose parameter is normalized so it's strange now. I will update the function that calculates the projection matrix in a few weeks.

**Comment by BigDataHa** [comment]
Thanks, and How to do texture Sampling, will you release the texture sampling code?

**Comment by Fanziapril** [comment]
Our paper is related to a patent and in this case, we cannot release the training code, including the texture sampling part. We have described the details in the paper and you can implement this part.

**Comment by enjoybo** [comment]
Hello, I found that there are seven pose parameters, which are different with your paper. I follow the order of pose parameters in paper but I got the wrong result when I rendered the model to image.

**Comment by dafuny** [comment]
> Hello, I found that there are seven pose parameters, which are different with your paper. I follow the order of pose parameters in paper but I got the wrong result when I rendered the model to image.

Hi,have you gotten some reasonable results?

**Comment by enjoybo** [comment]
Not yet, using the parameters of the output directly gets the wrong result, maybe the author have normalized the pose parameters.------------------ 原始邮件 ------------------
发件人: "yuefurong97"<notifications@github.com>
发送时间: 2019年7月18日(星期四) 下午4:16
收件人: "Fanziapril/mvfnet"<mvfnet@noreply.github.com>;
抄送: "enjoybo"<626341449@qq.com>;"Comment"<comment@noreply.github.com>;
主题: Re: [Fanziapril/mvfnet] pose parameters (#2)


 
Hello, I found that there are seven pose parameters, which are different with your paper. I follow the order of pose parameters in paper but I got the wrong result when I rendered the model to image.
  
Hi,have you gotten some reasonable results?
 
&mdash;

**Comment by Agent-INF** [comment]
Hey @Fanziapril ! Is there any update?
What are the seven pose parameters meaning? I have tried replacing three ruler angles with four quaternions, but it seems not working.
Looking forward to your reply!

**Comment by Fanziapril** [comment]
Hi all, the pose parameters have been updated.
